evm/v0.5/src/helpers.ts

```diff
@@ -538,7 +604,31 @@ export function hexToBuf(hexstr: string): Buffer {
   return Buffer.from(stripHexPrefix(hexstr), 'hex')
 }
 
-type Hash = ReturnType<typeof ethers.utils.keccak256>
+type Hash = ReturnType<typeof utils.keccak256>
+
+export async function newServiceAgreement(
```

Comment:
Could you refactor line 44 in `BasicServiceAgreementConsumer.test.ts` to use this new function?